http/headers/Link.json

```diff
@@ -12,7 +12,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": null
+              "version_added": true
```

Comment:
```suggestion
              "version_added": "1"
```

Reply:
I almost missed this because you left a review comment rather than a suggestion comment...

Reply:
I did both, though I meant to add this to the review.  🙃